server/core/db/boltdb.go

```diff
@@ -334,8 +334,9 @@ func (b *BoltDB) UpdatePullWithResults(pull models.PullRequest, newResults []mod
 				statuses = append(statuses, b.projectResultToProject(r))
 			}
 			newStatus = models.PullStatus{
-				Pull:     pull,
-				Projects: statuses,
+				Pull:               pull,
+				Projects:           statuses,
+				LastEventTimestamp: eventTimestamp,
```

Comment:
Why not just do `time.Now()` here? Technically, we don't care about the event timestamp in this context, only in the context of an event coming in

Also, let's call this `UpdatedAt` or something with Updated in it.